### PR TITLE
Development

### PR DESCRIPTION
**Pull request by asiimwedismas**
Please review this code so we can merge it to the master branch

We couldn't;t figure out how to host a live demo, maybe that's something u will shade some light one

### CODE REVIEW

**Comment by addod19** [comment]
> Please review this code so we can merge it to the master branch
> 
> We couldn't;t figure out how to host a live demo, maybe that's something u will shade some light one

Navigate to the index.html page on your Github repo and copy the link to this link http://raw.githack.com/. The link will generate development and production link. You can use the development link for your README file live link

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        </form>
+        <div class="float-right">
+            <span id="upload-button"></span>
+            <button id="sign-up-button">Sign up</button>
+        </div>
+    </header>
+
+    <main>
+
+        <aside id="thumbnails-aside">
```
Comment (addod19) [inline index.html:31]: The ```<aside>``` is not to be nested in the main. The aside should take the video previews and the main should have the main video together with the other contents

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi **TEAM**, you have done a great job with this project 👍 Let us improve our project before we move ahead. Kindly address the comments below: 😄 
* The nav is missing 3 images at the rightmost part of the nav
im
* Kindly edit the PR so it describes the purpose of the project or what exactly you did
* We have 1 warning returned by w3validator
* The main video is not playing. It should play
* The overall UI of the project needs improvement. Check this [link](youtube.com) as a reference

Make these changes and submit another code review 🙂 
_Hapy coding_ 👨‍💻 
Daniel Larbi Addo

**Comment by asiimwedismas** [comment]
Worked on the issues raised @addod19

**Comment by asiimwedismas** [comment]
The video doesn't seem to be loading on githack but it does on our development machines 
I have also noticed the youtube logo in footer isn't loading but once again, it loaded fine on our development machines

**Review by MarioDena — APPROVED** [review]
Congratulations on implementing the requested changes!

## Project approved! 🥇 

## Code Review by Mario Enrique Delgado Naranjo

**Pull request merged**